infrastructure/main.tf

```diff
@@ -38,9 +49,9 @@ module "citizen-frontend" {
   location = "${var.location}"
   env = "${var.env}"
   ilbIp = "${var.ilbIp}"
-  is_frontend  = true
+  is_frontend = "${var.env != "preview" ? 1: 0}"
   subscription = "${var.subscription}"
-  additional_host_name = "${var.external_host_name}"
+  additional_host_name = "${var.env != "preview" ? var.external_host_name : "null"}"
```

Comment:
Is the `"null` string expected downstream?

Reply:
https://github.com/hmcts/moj-module-webapp/blob/master/variables.tf#L69